File: app/agent/skills/container_planner.go
package skills

import (
	"math"
	"strconv"
)

func PlanContainer(totalCBM float64) string {
	switch {
	case totalCBM > 0 && totalCBM < 15:
		return "LCL 拼箱"
	case totalCBM <= 28:
		return "1\u00d720GP"
	case totalCBM <= 58:
		return "1\u00d740GP"
	case totalCBM <= 68:
		return "1\u00d740HQ"
	default:
		return strconv.Itoa(int(math.Ceil(totalCBM/68))) + "\u00d740HQ"
	}
}

File: ifs-portal/src/utils/containerPlanner.ts
export function planContainer(totalCBM: number) {
  if (totalCBM <= 28) return '1×20GP'
  if (totalCBM <= 58) return '1×40GP'
  if (totalCBM <= 68) return '1×40HQ'
  return `${Math.ceil(totalCBM / 68)}×40HQ`
}
